static/app/views/eventsV2/table/columnEditModal.spec.jsx

```diff
@@ -354,24 +484,29 @@ describe('EventsV2 -> ColumnEditModal', function () {
         },
         initialData
       );
-      expect(newWrapper.find('QueryField ArithmeticError')).toHaveLength(1);
-      expect(newWrapper.find('QueryField ArithmeticError').prop('title')).toBe(
-        'Division by 0 is not allowed'
-      );
 
-      const field = newWrapper.find('QueryField input[type="text"]');
-      changeInputValue(field, '1+1+1+1+1+1+1+1+1+1+1+1');
-      newWrapper.update();
-      field.simulate('blur');
+      userEvent.hover(await screen.findByTestId('arithmeticErrorWarning'));
+      const tooltipEl = await screen.findByText('Division by 0 is not allowed');
+      expect(tooltipEl).toBeInTheDocument();
 
-      expect(newWrapper.find('QueryField ArithmeticError').prop('title')).toBe(
-        'Maximum operators exceeded'
-      );
+      const input = screen.getAllByRole('textbox')[0];
+      expect(input).toHaveValue('1 / 0');
+
+      userEvent.clear(input);
+      userEvent.type(input, '1+1+1+1+1+1+1+1+1+1+1+1');
+      userEvent.click(document.body);
+
+      await waitFor(() => expect(input).toHaveValue('1+1+1+1+1+1+1+1+1+1+1+1'));
+
+      userEvent.hover(await screen.findByTestId('arithmeticErrorWarning'));
```

Comment:
```suggestion
      userEvent.hover(screen.getByTestId('arithmeticErrorWarning'));
```

do we need a `await` here?